### PR TITLE
Scroll restoration for button links

### LINKED ISSUE #806: Update Links due to navigation issue for Community Page
Currently the buttons on the community page take you to the desired page but to the middle or bottom of the desired page and never to the top of the desired page as shown in the screenshots. This arises due to error in navigation.

Kindly request you to assign me this issue so that i can rectify the same.

Screenshot 2023-10-04 at 3 04 44 PM
Screenshot 2023-10-04 at 3 04 28 PM

### PR DESCRIPTION
**Pull request by Smartmind12**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [X] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR rectifies the navigation issue on button click and opens the button on the same page . 

## Related Tickets & Documents

Resolves #806 #807 #808 

## Mobile & Desktop Screenshots/Recordings

On clicking the button the url opens on the same tab this rectifies the scrolling issue.

On clicking the contributing button in the midddle of the page opens the url of the contribution page in the same tab and at the top of the page.
Screenshot 2023-10-07 at 10 59 39 AM

Screenshot 2023-10-07 at 10 59 48 AM

## Added code snippets?
- [X] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [X] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [X] 🙅 no documentation needed

### CODE REVIEW

File: site/src/components/Button.jsx [hunk 1]
```diff
@@ -63,13 +63,15 @@ const Button = ({
       {linkContent}
     </a>
   ) : (
-    <Link
-      href={url}
+    <a
       className="dark w-[fit-content] no-underline text-[18px] font-medium"
+      rel="noopener noreferrer"
```
Comment (angiejones) [inline site/src/components/Button.jsx:68]: can you remove the `rel` attribute? knowing the referring internal page can be valuable for analytics and such

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/src/components/Button.jsx (hunk 1 above)
Comment (Smartmind12) [reply site/src/components/Button.jsx]: @angiejones Yes i have made the change!

**Comment by Smartmind12** [comment]
@angiejones I have removed the 'ref' attribute as suggested by you!

**Review by angiejones — APPROVED** [review]

**Comment by angiejones** [comment]
thank you @Smartmind12 
@EbonyLouis hacktoberfest

**Pull request merged**